### PR TITLE
Changed 1 to 2

### PR DESCRIPTION
**Pull request by gancearroman**
Changed 1 to 2

### CODE REVIEW

File: first_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+3
```
Comment (gh3rtzz) [inline first_file.txt]: Nice. Change it to 4 now

**Review by gh3rtzz — APPROVED** [review]
Nice, go ahead and merge it

**Comment by gancearroman** [comment]
Changed 1 to 4

**Comment by gancearroman** [comment]
> Nice, go ahead and merge it

I did it. Thank you for your support.

**Pull request merged**